### DISCUSSION TITLE
Mount a snapshot and browse the files in my linux FS

### DISCUSSION THREAD

**Discussion by zaggash** [garethgeorge/backrest#215] [General]
After the repo is setup, and a snapshot is uploaded, is there a way to mount the snapshot on a folder through the UI to browse ?

I had a feeling this mount path was exactly for that purpose:
` /MY-REPOS:/repos # (optional) mount your restic repositories somewhere in the filesystem.
`

**Comment by garethgeorge** [comment]
Backrest provides a built in snapshot browser
![image](https://github.com/garethgeorge/backrest/assets/7906572/af770629-26a0-4812-aa77-07a1c9341f09)
you can find this in the UI after running a backup (or indexing snapshots) but it doesn't do filesystem mounts the way restic will

**Comment by zaggash** [comment]
Alright.
I found this one already thanks.

So I guess the only purpose of :
`/MY-REPOS:/repos # (optional) mount your restic repositories somewhere in the filesystem.`

is to create a local repository for local backups.

Thanks.
